internal/mode/static/state/graph/httproute.go

```diff
@@ -806,16 +809,16 @@ func validateFilterRewrite(
 }
 
 func validateFilterHeaderModifier(
+	filterType v1.HTTPRouteFilterType,
```

Comment:
Do we need to pass the filterType in? Can we just add the filter type to the path when we pass it to function:

```
validateFilterHeaderModifier(validator, filter.ResponseHeaderModifier, filterPath.Child("responseHeaderModifier"))
```

Reply:
Is there any elegant way to retrieve the child of the `filterPath` (i.e. `"responseHeaderModifier"`) so that I can use it in `panicForBrokenWebhookAssumption(errors.New(filterTypeStr + " cannot be nil"))`?

Reply:
Updated https://github.com/nginxinc/nginx-gateway-fabric/pull/1494/commits/2213a64d0182350664711d667ad00914995716e6. Note that `string(filter.Type)` is `RequestHeaderModifier` instead of `requestHeaderModifier`. I am not sure whether the capitalization of the first character matters or not.